DynamicBlurView/DynamicBlurView.swift

```diff
@@ -135,17 +135,22 @@ open class DynamicBlurView: UIView {
 
     private func snapshotImage(for layer: CALayer, conversion: Bool) -> UIImage? {
         let rect = blurLayerRect(to: layer, conversion: conversion)
-        guard let context = CGContext.imageContext(with: quality, rect: rect, opaque: isOpaque) else {
-            return nil
+        
+        var format = UIGraphicsImageRendererFormat()
+        format.scale = quality.imageScale
+        format.opaque = isOpaque
+        
+        let renderer = UIGraphicsImageRenderer(size: bounds.size, format: format)
+        let image = renderer.image { context in
+            let cgContext = context.cgContext
+            
+            cgContext.translateBy(x: -rect.origin.x, y: -rect.origin.y)
```

Comment:
question - why we need `translate`?

Reply:
I think I confused it for this function : 
`
static func imageContext(with quality: CaptureQuality, rect: CGRect, opaque: Bool) -> CGContext? {
        UIGraphicsBeginImageContextWithOptions(rect.size, opaque, quality.imageScale)
        guard let context = UIGraphicsGetCurrentContext() else {
            return nil
        }

        context.translateBy(x: -rect.origin.x, y: -rect.origin.y)
        context.interpolationQuality = quality.interpolationQuality

        return context
    }
`

Reply:
from this file : https://github.com/myxplor/DynamicBlurView/blob/version/2.0.2/DynamicBlurView/CGContext%2BCGImage.swift 
So I think I should replace that one too